### ISSUE TITLE
[Tuning] Zscaler.ZIA.TrustModification

### ISSUE THREAD

**Issue by zestysoft** [panther-labs/panther-analysis#1877]
## Rule ID

Zscaler.ZIA.TrustModification

## Description

This rule has a bug that causes false positive alerts when both `samlEnabled` fields are missing from an event.

The rule uses different default values for `saml_enabled_pre` and `saml_enabled_post`:

```python
saml_enabled_pre = event.deep_get(
    "event", "preaction", "samlEnabled", default="<PRE_SAML_STATUS_NOT_FOUND>"
)
saml_enabled_post = event.deep_get(
    "event", "postaction", "samlEnabled", default="<POST_SAML_STATUS_NOT_FOUND>"
)
if (
    action == "UPDATE"
    and category == "ADMINISTRATOR_MANAGEMENT"
    and saml_enabled_pre != saml_enabled_post
):
    return True
```

When both `samlEnabled` fields are missing from an event:
- `saml_enabled_pre` = `"<PRE_SAML_STATUS_NOT_FOUND>"`
- `saml_enabled_post` = `"<POST_SAML_STATUS_NOT_FOUND>"`

Since these two strings are different, the inequality check `saml_enabled_pre != saml_enabled_post` evaluates to `True`, causing false positive alerts for any UPDATE event in the ADMINISTRATOR_MANAGEMENT category that lacks these fields.
